integration-tests/babel-plugin-jest-hoist/__tests__/integration.test.js

```diff
@@ -44,6 +45,7 @@ jest.mock('../__test_modules__/e', () => {
     },
   };
 });
+jest.mock('../__test_modules__/jest-backticks');
```

Comment:
should this use backticks?

Reply:
You're absolutelly right, I was testing if the backticks were really working and forgot to update this. Thanks :)